fusor-ember-cli/app/models/deployment-plan.js

```diff
@@ -3,133 +3,22 @@ import DS from 'ember-data';
 
 export default DS.Model.extend({
   name: DS.attr('string'),
-  uuid: DS.attr('string'),
-  roles: DS.hasMany('deployment-role', {inverse: null, async: false}),
-  parameters: DS.hasMany('deployment-plan-parameter', {inverse: null, async: false}),
+  parameters: DS.attr(),
+  roles: DS.attr(),
```

Comment:
Now that I look at this, I'll check and make sure this is correct.  Thanks, nice catch.